pages/auth/login.vue

```diff
@@ -322,11 +323,11 @@ export default {
                 />
               </div>
               <div class="">
-                <LabeledInput
+                <Password
                   ref="password"
-                  v-model="password"
-                  type="password"
+                  v-model.trim="password"
```

Comment:
The password shouldn't be trimmed. I'm not 100% sure whitespace would be accepted, but we shouldn't morph the user's input

Reply:
Done